Dockerfile

```
@@ -1,5 +1,5 @@
 # Container image that runs your code
-FROM openjdk:latest
+FROM eclipse-temurin:latest
     
 #RUN yum install curl
```

@saputhebeast the new base image is missing cURL and the `postinstall.sh` script will fail as a result.
I suggest we add it here, and since this image is Ubuntu based we need something like this:

```suggestion
RUN apt-get update \
    && apt-get install -y --no-install-recommends curl \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/*
```